openstudiocore/src/osversion/VersionTranslator.cpp

```diff
@@ -2550,6 +2550,7 @@ std::string VersionTranslator::update_1_7_4_to_1_7_5(const IdfFile& idf_1_7_4, c
 
 std::string VersionTranslator::update_1_8_3_to_1_8_4(const IdfFile& idf_1_8_3, const IddFileAndFactoryWrapper& idd_1_8_4)
 {
+  LOG(Error,"Here is the model" << idf_1_8_3);
```

Comment:
yes a mistake. I was doing some testing to figure out what the heck was going on.